index.html

```diff
@@ -10,7 +10,7 @@
     <meta name="theme-color" content="#FFB954">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="description" content="Loom is a next-generation Minecraft server software with a modern and extensive plugin API. It aims to replace projects like Spigot and CraftBukkit.">
-    <meta name="keywords" content="Minecraft, Spigot, server, plugins, api, open-source">
+    <meta name="keywords" content="Minecraft, Spigot, Bukkit, CraftBukkit, Paper, Tuinity, Yatopia, Purpur, Glowstone, Minestom, SpongeVanilla, Akarin, server, plugins, plugin, api, open-source, Lithium">
```

Comment:
thats false advertising!!!

Reply:
WHAT DO YOU MEAN??????????/

Reply:
Loom has nothing to do with Bukkit, CraftBukkit, Paper, Tuinity, Yatopia, Purpur, Glowstone, Minestom, SpongeVanilla, Akarin! Especially not Lithium.

Reply:
They already had Spigot in there, deal with it :sunglasses:

Reply:
no. spigot shouldn't be in there in the first place.

Reply:
Spigot is in there, because the initial goal of the project is to "replace" / provide a better solution than bukkit/spigot. The other projects however are not part of that goal at all, so should not be in there.

As of removing spigot from the list, maybe... But I honestly cannot be bothered, this is just a simple placeholder website and will be changed in the future.

Reply:
ok. that makes sense I guess